as2/server/player.md

```diff
@@ -0,0 +1,98 @@
+# Player
+##### Messages pertaining to the player actions.
+---
+### Send Frame
+Sent to all penguins in a room to update a player frame. 
+
+`sf`
+
+|Argument|Type|Variadic?|
+|---|---|---|
+|penguinID|Unsigned integer|No|
+|frame|Unsigned integer|No|
+
+##### Example
+`%xt%sf%0%123%5%`
+
+### Send Move
+Sent to all penguins in a room to update a player location. 
```

Comment:
Maybe add a note in the description that the client will animate waddling from point A to B on its own accord.